db/migrate/20230501161349_create_income_limits_tables.rb

```diff
@@ -0,0 +1,90 @@
+class CreateIncomeLimitsTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :std_counties do |t|
+      t.string :name, null: false
+      t.integer :countynumber, null: false
+      t.string :description, null: false
+      t.integer :state_id, null: false
+      t.integer :version, null: false
+      t.datetime :created, null: false
+      t.datetime :updated
+      t.string :createdby
+      t.string :updatedby
+    end
+    create_table :gmt_thresholds do |t|
+      t.integer :effectiveyear, null: false
+      t.string :statename, null: false
+      t.string :countyname, null: false
+      t.integer :fips, null: false
+      t.integer :trhd1, null: false
+      t.integer :trhd2, null: false
+      t.integer :trhd3, null: false
+      t.integer :trhd4, null: false
+      t.integer :trhd5, null: false
+      t.integer :trhd6, null: false
+      t.integer :trhd7, null: false
+      t.integer :trhd8, null: false
+      t.integer :msa, null: false
+      t.string :msaname
+      t.integer :version, null: false
+      t.datetime :created, null: false
+      t.datetime :updated
+      t.string :createdby
+      t.string :updatedby
+    end
+    create_table :std_incomethresholds do |t|
+      t.integer :income_threshold_year, null: false
+      t.integer :exempt_amount, null: false
+      t.integer :medical_expense_deductible, null: false
+      t.integer :child_income_exclusion, null: false
+      t.integer :dependent, null: false
+      t.integer :add_dependent_threshold, null: false
+      t.integer :property_threshold, null: false
+      t.integer :pension_threshold
+      t.integer :pension_1_dependent
+      t.integer :add_dependent_pension
+      t.integer :ninety_day_hospital_copay
+      t.integer :add_90_day_hospital_copay
```

Comment:
**Nitpick:** `ninety` is written out on the above line but is `90` here.

Reply:
good call, ty